azure-pipelines.yml

```diff
@@ -37,7 +37,7 @@ variables:
   - name: ARTIFACTORY_API_KEY
     value: $[variables.ARTIFACTORY_PROMOTER_ACCESS_TOKEN]
   - name: IS_RELEASE_BRANCH
-    value: ${{ or(eq(variables['Build.SourceBranch'], 'refs/heads/master'), startsWith(variables['Build.SourceBranch'], 'refs/heads/branch-')) }}
+    value: "true"
```

Comment:
Educational: why are we setting IS_RELEASE_BRANCH always to `true`?

Reply:
This is the debug commit that I will drop before merging. It's set to true so I can test that the files are indeed signed. I'm sorry, I forgot to mention.